bin/stacks/routing-caching-stack.ts

```diff
@@ -70,9 +91,41 @@ export class RoutingCachingStack extends cdk.NestedStack {
 
     this.poolCacheBucket.grantReadWrite(poolCachingLambda);
 
+    const ipfsPoolCachingLambda = new aws_lambda_nodejs.NodejsFunction(
```

Comment:
Fine for testing but when we merge we should only really create this resource if `stage == beta` (and eventually prod). It would just fail locally since we don't have permissions to access the route53 role anyway